CHANGELOG.md

```diff
@@ -1,8 +1,54 @@
 # Release 2.0
 
-## What's new
+## Table of contents
 
-This is a brief development summary for this release.
+* [2.1.0](#whats-new-in-210)
+* [2.0.0](#whats-new-in-200)
+
+## What's new in 2.1.0
+
+Release **2.1.0** is a compatibility release for the core **Magento 2.3.1** release.
+
+Notable changes include:
+
+* Updated GraphQL queries
+* Magento GraphQL query validation tool for buildpack
```

Comment:
The plugin is in its own directory (and outside of `pwa-buildpack`), but it is still part of the build process (runs when Venia gets built). Should this be updated?